ocfweb/login/templates/ocf/login.html

```diff
@@ -1,26 +1,41 @@
 {% extends "base.html" %}
-{% block title %}OCF Login{% endblock %}
+{% load bootstrap %}
 
 {% block content %}
+    <div class="ocf-content-block">
+        <div class="row">
+            <div class="col-sm-8">
+                {% if error %}
+                    <div class="alert alert-danger" role="alert">
+                        {{ error }}
+                    </div>
+                {% endif %}
 
-{% if error %}
-<div class="login-error">
-{{ error|linebreaks }}
-</div>
-{% endif %}
-
-<style type="text/css">
-.login-error{
-    border:1px solid #850e0e;
-    background-color:#f55858;
-}
-</style>
-
-<p>The page you're attempting to access requires you to log in. Please enter your OCF username and password.</p>
-<form action="{% url 'login' %}" method="post">
-    {% csrf_token %}
-    {{ form.as_p }}
-    <input type="submit" value="Login" />
-</form>
-
+                <p>
+                    Sorry! Before you can proceed, we need to know who you are.
+                    Please enter your OCF username and password below to continue.
+                </p>
+                <form action="{% url 'login' %}" method="post">
+                    {% csrf_token %}
+                    <div style="max-width: 300px;">
```

Comment:
Ok, that makes sense, forms will likely be different sizes, so it shouldn't be limited. We can always change this if we encounter later on that we only have a couple common sizes or that kind of thing, but for now I would agree that it can just be left.